packages/esm-patient-vitals-app/src/vitals/vitals-overview.component.tsx

```diff
@@ -217,6 +217,7 @@ const VitalsOverview: React.FC<VitalsOverviewProps> = ({ patientUuid, pageSize,
                 <div ref={contentToPrintRef}>
                   <PrintComponent subheader={headerTitle} patientDetails={patientDetails} />
                   <PaginatedVitals
+                    patientUuid={patientUuid}
```

Comment:
This is one of those points where the multiple React trees causes things to not work as we might like (only the patient chart component actually parses the URL. Maybe this would be a good case for modifying the component decorator to put the props passed to an extension in a React context and then we could create a `useExtensionProps()` hook to retrieve them?

Reply:
Only if it's more performant than drilling the prop down. Otherwise, it sounds like a lot of overhead.

Reply:
Hmmm... I think it'd be slightly more _sane_ than deep prop drilling, but it's unlikely to be more performant. Technically, I think that `usePatient()` does parse the URL correctly for the chart, but I also don't think it's better to do that multiple times.

I'm going to go with: this is one of the things that the AppContext feature is intended to fix, so that you can just call `usePatient()` and it will contextually "know" which patient is active (at least in the chart!).

Reply:
Sounds good!